packages/c/claws-mail/files/org.claws_mail.Claws-Mail.appdata.xml

```diff
@@ -0,0 +1,359 @@
+	</ul>
+	<p>Bug fixes:</p>
+	<ul>
+          <li>Bug 1920, 'No automatic NNTP filtering'</li>
+          <li>Bug 2045, 'address book blocks focus on email window'</li>
+          <li>Bug 2131, 'Focus stealing after mail check'</li>
+          <li>Bug 2627, 'Filtering does not work on NNTP'</li>
+          <li>Bug 3070, 'misbehaving text wrapping when URL chars are present'</li>
+          <li>Bug 3838, 'Canceled right-click on message list leaves UI in inconsistent state'</li>
+          <li>Bug 3977, 'Fix crashes when some external APIs fail'</li>
+          <li>Bug 3979, 'Hang (with killing needed) during action which extracts attachments'</li>
+          <li>Bug 4029, 'segfault after deleting message in a window'</li>
+          <li>Bug 4031, 'fingerprint in SSL/TLS certificates for ... (regress error)'</li>
+          <li>Bug 4037, 'Fix some small issues'</li>
+          <li>Bug 4142, 'Translation error on Russian'</li>
+          <li>Bug 4145, 'proxy server for sending doesn't work'</li>
+          <li>Bug 4155, 'remember directory of last saving'</li>
+          <li>Bug 4166, 'corrupted double-linked list'</li>
+          <li>Bug 4167, 'Max line length exceeded when forwarding mail'</li>
+          <li>Bug 4188, 'STL file is sent not as an attachment but as its base64 representation in plaintext'</li>
+          <li>CID 1442278, 'impossible to trigger buffer overflow'</li>
+          <li>Make key accelerators from menu work in addressbook window</li>
+          <li>Save checkbox choices of display/summaries/defaults prefs</li>
+          <li>Do not throw an error when cancelling 'Save email as...'</li>
+          <li>Occasional crash on drag'n'drop of msgs</li>
+          <li>Possible stack overflow in vcalendar's Curl data handler</li>
+          <li>Crash when LDAP address source is defined in index, but LDAP support is disabled</li>
+          <li>Crash in Fancy plugin if one of the MIME parts has no Content-ID</li>
+          <li>A few small memory leaks in scan_mailto_url()</li>
+          <li>Configure script for rare cases where python is not installed</li>
+          <li>Incorrect charset conversion in sc_html_read_line()</li>
+          <li>Markup in 'key not fully trusted' warning in pgpcore</li>
+          <li>Use after free in rare code path in rssyl_subscribe()</li>
+          <li>Several memory leaks</li>
+          <li>Verify_folderlist_xml() for fresh starts</li>
+          <li>Printf formats for size_t and goffset arguments</li>
+          <li>Alertpanel API use in win32 part of mimeview.c</li>
+          <li>Pid handling in debug output of kill_children_cb()</li>
+          <li>Incorrect pointer arithmetic in w32_filesel.c</li>
+        </ul>
+      </description>
+    </release>
+  </releases>
+
+  <content_rating type="oars-1.0">
+    <content_attribute id="social-contacts">intense</content_attribute>
+  </content_rating>
+  <update_contact>danny_AT_dannyvanheumen.nl</update_contact>
```

Comment:
Remove update contact, we don't want user to contact the dev directly if they need support.